OpenRA.Mods.Common/Traits/BotModules/BotModuleLogic/BaseBuilderQueueManager.cs

```diff
@@ -364,15 +389,56 @@ ActorInfo ChooseBuildingToBuild(ProductionQueue queue)
 			}
 
 			// Too spammy to keep enabled all the time, but very useful when debugging specific issues.
-			// AIUtils.BotDebug("{0} couldn't decide what to build for queue {1}.", queue.Actor.Owner, queue.Info.Group);
+			// AIUtils.BotDebug($"{queue.Actor.Owner} couldn't decide what to build for queue {queue.Info.Group}.");
 			return null;
 		}
 
+		CPos? ChooseBarrierLocation(ActorInfo wallInfo)
```

Comment:
Can we rename this to `wallActorInfo`? Using just `wallInfo` is Imho somewhat confusing considering we usually use `<traitname>Info` (for example `buildingInfo`) but this is an `ActorInfo` and not the info of a `Wall` trait.